README.md

```diff
@@ -131,6 +131,34 @@ RAILS_ENV=production bundle exec rake m2c_exist_all_storage_roots[profile]
 ```
 this will generate a log at, for example, ```log/profile_check_existence_for_all_storage_roots2017-12-11T14:25:31-flat.txt```
 
+### Run Catalog to Moab existence check for a single root and for all storage roots
+
+- To run rake tasks below, give the name of the moab storage_root (e.g. from settings/development.yml) and the last checked before date as an argument.
```

Comment:
we need a better explanation of "last checked before date" ... 

we're also going to want an exemplar date in the right format, not 'last_checked_b4_date. @tallenaz will know the correct date format.

How about:
```
To run rake tasks below, give a date (timestamp?  whatever tony says)** and the name of the moab storage_root (e.g. from settings/development.yml) as arguments.

The (date/timestamp) argument is a threshold:  it will run the check on all catalog entries which last had a version check BEFORE the argument.   It should be in the format xxxx.
```

(also ask @tallenaz if it is BEFORE or  BEFORE or EQUAL TO)